pkgs/games/minecraft/default.nix

```diff
@@ -1,12 +1,11 @@
-{ lib, stdenv
+{ lib
+, stdenv
+, buildFHSEnv
 , fetchurl
+, fetchzip
 , nixosTests
 , copyDesktopItems
 , makeDesktopItem
-, makeWrapper
-, wrapGAppsHook3
-, gobject-introspection
-, jre # old or modded versions of the game may require Java 8 (https://aur.archlinux.org/packages/minecraft-launcher/#pinned-674960)
```

Comment:
I have a question, does it work with old minecraft versions as per the above warning?